packages/autorest.typescript/src/restLevelClient/transforms/transformOptions.ts

```diff
@@ -17,9 +17,9 @@ export function transformOptions(model: CodeModel): RLCOptions {
     dependencyInfo,
     productDocLink,
     azureArm,
-    flavor,
+    flavor
   } = getAutorestOptions();
-  const options: RLCOptions = {};
+  const options: RLCOptions = { moduleKind: "cjs" };
```

Comment:
There should be no reason to emit CJS since we can emit for that using tshy as well as for `react-native`